catalog/content/core/content-core-impl/src/main/java/ddf/content/impl/ContentFrameworkImpl.java

```diff
@@ -291,7 +322,24 @@ public UpdateResponse update(final UpdateRequest updateRequest, Request.Directiv
 
             LOGGER.info("Updating content repository for content item: " + itemToUpdate.getId());
             try {
+                for (final PreUpdateStoragePlugin plugin : preUpdateStoragePlugins) {
+                    try {
+                        updateRequest = plugin.process(updateRequest);
+                    } catch (PluginExecutionException e) {
```

Comment:
Ahhh yeah nevermind. We do catch exception on this  block below where it shows here so should be good.